common/djangoapps/course_modes/models.py

```diff
@@ -608,6 +609,9 @@ def is_white_label(cls, course_id, modes_dict=None):
         if modes_dict is None:
             modes_dict = cls.modes_for_course_dict(course_id)
 
+        if configuration_helpers.is_site_configuration_enabled():
+            return configuration_helpers.get_value('show_ecommerce_reports', False)
```

Comment:
In that way, Reports section will be displayed on value of `show_ecommerce_reports` flag from SiteConfiguration for all courses ?